src/main/kotlin/lotto/domain/LottoMachine.kt

```diff
@@ -0,0 +1,20 @@
+package lotto.domain
+
+class LottoMachine {
+    fun createLottos(amount: Amount): List<Lotto> {
+        val number = amount.divide(LOTTO_PRICE).toInt()
+        return List(number) { createAutoLotto() }
+    }
+
+    private fun createAutoLotto() =
+        Lotto(
+            LottoNumber.cached.shuffled()
+                .take(LOTTO_NUMBERS_SIZE)
+                .map(LottoNumber::from),
```

Comment:
Using function reference 👍

I also like using function references when I can, check out this article to learn more about it!

https://proandroiddev.com/kotlin-lambda-vs-method-reference-fdbd175f6845
> To sum up, the difference between lambda and method reference is that when we use method reference, the instance of variable, the method of which we use, is fixed at the moment the reference is created, not when it is called, unlike what happens when lambda is used.